lms/services/lti_h.py

```diff
@@ -48,11 +48,12 @@ def sync(self, groupings: List[Grouping], group_info_params: dict):
 
         self._h_api.execute_bulk(commands=self._yield_commands(groupings))
 
-        # Keep a note of the groups locally for reporting purposes.
-        for grouping in groupings:
-            self._group_info_service.upsert_group_info(
-                grouping=grouping, params=group_info_params
-            )
+        if group_info_params:
```

Comment:
We don't need to keep track of these (sections/groups) in group info. We already have `Grouping` for that.